src/main/java/com/deavensoft/paketomat/mapper/CityMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.deavensoft.paketomat.mapper;
+
+import com.deavensoft.paketomat.center.model.City;
+import com.deavensoft.paketomat.city.dto.CityDto;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring", uses = CitiesMapper.class)
```

Comment:
Do we really need _uses_ attribute?

Reply:
Done.

Reply:
Checked the uses attribute, and we dont need it here.